### DISCUSSION TITLE
Ενδεικτικά εργαλεία ανάπτυξης

### DISCUSSION THREAD

**Discussion by mxidakis** [sdy60/syllabus#172] [Q&A] [answered]
Το ερώτημα αναφέρεται σε διαθέσιμα εργαλεία ανάπτυξης. Θα μπορούσε κάποιος να εξηγήσει σε ποιά εργαλεία αναφέρεται δίνοντας κάποια παραδείγματα;

**Comment by epidrome** [comment, marked as answer]
Η απάντηση σε αυτό το ερώτημα είναι το ζητούμενο οπότε πολύ πρόχειρα και ενδεικτικά ένα εργαλείο είναι το Apple Xcode, υπάρχουν πολλά εργαλεία, βιβλιοθήκες, τεχνικές για την παραγωγή εφαρμογών ανάλογα με τις παραδοχές που θα κάνετε για το κινήτο και διάχυτο σύστημα που σχεδιάζετε
